main.go

```diff
@@ -125,7 +125,7 @@ func processUpdate(topic, payload string) {
 	if _, ok := ignoreKeyMetrics[topic]; !ok {
 		if _, ok := counterKeyMetrics[topic]; ok {
 			//log.Printf("Processing counter metric %s with data %s", topic, payload)
-			processCounterMetric(topic, payload)
+			processGaugeMetric(topic, payload)
```

Comment:
Why I would change the counters to gauges??

Reply:
Counter metrics should processed as a counters.

Reply:
i agree but the problem is that `processCounterMetric()` uses `.Add()` and not `.Set()` (which makes sense obviously).
but i noticed that the payload basically is "hey, i am processing 42 messages per second".
so the metric would show `42`.

and the next time `processCounterMetric()` is called it adds the payload to the previous one.
so the metric now would show `84` etc.

if the payload was "hey, i processed 1337 messages since you last asked", the behaviour of `processCounterMetric()` would be correct.

Reply:
You are right. I will work in a fix to keep this metrics as a counter.